packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -57,14 +39,18 @@ const MarketPlacePage = () => {
     );
   };
 
-  const { status, data } = useQuery('list-plugins', () => fetchPlugins(notifyLoad), {
-    onError: () => {
-      toggleNotification({
-        type: 'warning',
-        message: { id: 'notification.error', defaultMessage: 'An error occured' },
-      });
-    },
-  });
+  const { status, data: pluginsResponse } = useQuery(
+    'list-plugins',
+    () => fetchPlugins(notifyLoad),
+    {
+      onError: () => {
+        toggleNotification({
+          type: 'warning',
+          message: { id: 'notification.error', defaultMessage: 'An error occured' },
```

Comment:
Can we improve the error message to indicate what went wrong? E.g.
```suggestion
          message: { id: 'notification.error', defaultMessage: 'List of available plugins could not be fetched' },
```

Otherwise the user might not know what actually went wrong ...